Sources/SwiftInspectorVisitors/Tests/ClassVisitorSpec.swift

```diff
@@ -1,413 +0,0 @@
-                public enum BarFooEnum {
-                  public class BarBarFooClass {}
-                }
-                public class FooFooClass {
-                  public class BarFooFooClass {}
-                }
-              }
-              """
-
-            try? VisitorExecutor.walkVisitor(
-              self.sut,
-              overContent: content)
-          }
-
-          it("finds FooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "FooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == nil
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooStruct") {
-            let matching = self.sut.innerStructs.filter {
-              $0.name == "BarFooStruct"
-                && $0.inheritsFromTypes.map { $0.asSource } == ["Equatable"]
-                && $0.parentType?.asSource == "FooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooStruct.BarBarFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "BarBarFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.BarFooStruct"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.FooFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "FooFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.FooFooClass.BarFooFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "BarFooFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.FooFooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooEnum") {
-            let matching = self.sut.innerEnums.filter {
-              $0.name == "BarFooEnum"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass"
-            }
-
-            expect(matching.count) == 1
-          }
-
-          it("finds FooClass.BarFooEnum.BarBarFooClass") {
-            let matching = self.sut.classes.filter {
-              $0.name == "BarBarFooClass"
-                && $0.inheritsFromTypes.map { $0.asSource } == []
-                && $0.parentType?.asSource == "FooClass.BarFooEnum"
-            }
-
-            expect(matching.count) == 1
-          }
-        }
-      }
-
-      context("visiting a code block with multiple top-level declarations") {
-        context("with multiple top-level classes") {
```

Comment:
This context was brought into the new file under a context named `with a top-level class declaration`